### ISSUE TITLE
Close all acwrite buffers on vim exit

### ISSUE THREAD

**Issue by timtyrrell** [hoschi/yode-nvim#7]
Restoring a session will restore the `acwrite` buffers in a non-awesome way. Is there a recommended method or a command that I can run to close all the buftype `acwrite` buffers when I exit vim or can one be added?

EDIT: 
Using https://github.com/kazhala/close-buffers.nvim I am able to run `:BDelete glob=yode*` and they all get closed. Although, when I create an autocmd using the same command I am getting some errors. I tried a few of the exit and leave events and seemed to get the same error each time. Any thoughts on it?

 `autocmd ExitPre * :BDelete glob=yode*`

<img width="1018" alt="Screen Shot 2022-02-12 at 2 28 12 PM" src="https://user-images.githubusercontent.com/100535/153729066-e1532bed-5c24-4437-b464-80f02a4505b5.png">

**Comment by timtyrrell** [comment]
I was able to add that to a `pre_save_cmds` in my session plugin, I am good.

**Comment by hoschi** [comment]
Saving and loading sessions is something I haven't implemented for floating windows and seditors yet, unfortunately.

**Comment by hoschi** [comment]
@timtyrrell Could you please post your fix in #2, for the other users?

**Comment by timtyrrell** [comment]
@hoschi Sorry I didn't post earlier because I leverage plugins for the workflow, so it is surely not a solution for the masses but works for me. The main part is the [close-buffers glob feature](https://github.com/kazhala/close-buffers.nvim#glob) and then having [the session plugin](https://github.com/rmagatti/auto-session#-command-hooks) use the pre-save callback to run the command:

```
call plug#begin('~/.config/nvim/plugged')

Plug 'nvim-lua/plenary.nvim'
Plug 'hoschi/yode-nvim'
Plug 'kazhala/close-buffers.nvim'
Plug 'rmagatti/auto-session'

call plug#end()

lua << EOF

require('yode-nvim').setup {}
require('auto-session').setup {
  pre_save_cmds = {"BDelete glob=yode*"}
}

EOF
```
